bin/gcs_cli.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Manipulate emulated GCS storage.
+
+# Usage: ./bin/gcs_cli.py CMD
+
+import os
+
+import click
+import requests
+import urllib3
+
+from google.auth.credentials import AnonymousCredentials
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+base_url = os.environ["STORAGE_EMULATOR_HOST"]
+
+
+def get_client():
+    # Create a session that is OK talking over insecure HTTPS
+    weak_https = requests.Session()
+    weak_https.verify = False
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+    return storage.Client(
+        credentials=AnonymousCredentials(), project="test", _http=weak_https
+    )
+
+
+@click.group()
+def gcs_group():
+    """Local dev environment GCS manipulation script"""
+
+
+@gcs_group.command("create")
+@click.argument("bucket_name")
+def create_bucket(bucket_name):
```

Comment:
I'm not passing in the `ctx` object here as in `bin/s3_cli.py`, as I don't believe it is needed, though my understand per the `click` docs is fuzzy: https://click.palletsprojects.com/en/8.1.x/complex/#contexts . Is this needed?

Reply:
I add ctx because I often use `ctx.exit(1)` in error situations and I'm just used to setting up click command code that way. If you don't need to do that or use other things in `ctx`, then you don't need it.